x-pack/packages/security-solution/distribution_bar/README.md

```diff
@@ -0,0 +1,10 @@
+# Security Solution's Distribution Bar
+
+The DistributionBar component visually represents data distribution, such as critical, high, medium, and low alerts.
+
+## Storybook
+
+General look of the component can be checked visually running the following storybook:
+`yarn storybook security_solution_packages`
```

Comment:
Thank you for adding the documentation about the Distribution Bar and including instructions on how to run the Storybook!